tests/rhui3_tests/test_CLI.py

```diff
@@ -0,0 +1,112 @@
+#! /usr/bin/python -tt
+
+import nose, stitches, logging, yaml
+
+from rhui3_tests_lib.rhuimanager import *
+from rhui3_tests_lib.rhuimanager_repo import *
+from rhui3_tests_lib.rhuimanagercli import *
+
+from os.path import basename
+
+logging.basicConfig(level=logging.DEBUG)
+
+connection=stitches.connection.Connection("rhua.example.com", "root", "/root/.ssh/id_rsa_test")
+
+with open('/tmp/rhui3-tests/tests/rhui3_tests/tested_repos.yaml', 'r') as file:
+    doc = yaml.load(file)
+
+yum_repo_name_1 = doc['CLI_repo1']['name']
+yum_repo_id_1 = doc['CLI_repo1']['id']
+yum_repo_name_2 = doc['CLI_repo2']['name']
+yum_repo_id_2 = doc['CLI_repo2']['id']
+
+custom_repo_name = "my_custom_repo"
+
+def setUp():
+    print "*** Running %s: *** " % basename(__file__)
+
+def test_01_initial_run():
+    '''Do an initial rhui-manager run to make sure we are logged in'''
+    RHUIManager.initial_run(connection)
+
+def test_02_check_empty_repo_list():
+    '''Check if the repolist is empty (interactively; not currently supported by the CLI)'''
+    nose.tools.assert_equal(RHUIManagerRepo.list(connection), [])
+
+def test_03_remove_existing_entitlement_certificates():
+    '''Clean up uploaded entitlement certificates'''
+    RHUIManager.remove_rh_certs(connection)
+
+def test_04_create_custom_repo():
+    '''Create a custom repo for further testing (interactively; not currently supported by the CLI)'''
+    RHUIManagerRepo.add_custom_repo(connection, custom_repo_name, entitlement="n")
```

Comment:
@RadekBiba can I list that repo after adding to check it's there?

Reply:
OK. It's not that critical because this is already tested elsewhere, and if this very creation fails, the following attempt to upload stuff to the repo will fail, but I'll add the test.

Reply:
I remember RHUI-manager allowed to add the same id, but in the end reported in error. As far as I remember now, there is a check on id in the most beginning. I don't know how rhui-CLI works, and if or how it calls RHUI-manager functions.